docs/src/threadpool.rst

```diff
@@ -52,15 +96,59 @@ Public members
 
 .. seealso:: The :c:type:`uv_req_t` members also apply.
 
+.. c:member:: uv_executor_submit_func uv_executor_t.submit
+
+.. c:member:: uv_executor_cancel_func uv_executor_t.cancel
+
+.. c:member:: uv_work_type uv_work_options_t.type
+
+    Type of request. Readonly.
+
+    ::
+
+        typedef enum {
+            UV_WORK_UNKNOWN = 0,
+            UV_WORK_FS,
+            UV_WORK_DNS,
+            UV_WORK_USER_IO,
+            UV_WORK_USER_CPU,
+            UV_WORK_PRIVATE,
+            UV_WORK_TYPE_MAX
```

Comment:
This makes adding a new entry a breaking change, right? I think that could be avoided by making `UV_WORK_TYPE_MAX` the return value of a function.

Reply:
I think you're right ABI-wise but am not certain. Can also be addressed by setting UV_WORK_TYPE_MAX=255.